Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -819,4 +819,7 @@ def SwapKeepAndDrop(l):
 if fastSim.isChosen():
     for _entry in [FEVTDEBUGHLTEventContent,FEVTDEBUGEventContent,RECOSIMEventContent,AODSIMEventContent,RAWAODSIMEventContent]:
         fastSimEC.dropSimDigis(_entry.outputCommands)
-    
+
+from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
+for _entry in [FEVTDEBUGEventContent,FEVTDEBUGHLTEventContent,FEVTEventContent]:
+  phase2_tracker.toModify(_entry, outputCommands = _entry.outputCommands + ['keep Phase2TrackerDigiedmDetSetVector_*_*_*'])
```

Comment:
I just migrate the customization function. Maybe I did it in the wrong way, but I am not an expert on this module, so I cannot say it just looking at the custom files [here](https://github.com/cms-sw/cmssw/pull/16181/files#diff-b82e46eea612ffe857aae0dd6c492d8aL48). Do you have any suggestion how to do that? @kpedro88 or @boudoul have others?